charter.html

```diff
@@ -207,24 +238,32 @@ <h3 id=specifications>Specifications</h3>
   </tbody>
 </table>
 
-<h3 id=other-work>Other work</h3>
-<p>None.
-<!--
-<table>
-  <thead>
-    <tr>
-      <th>Name
-      <th>Editors
-    </tr>
-  </thead>
-  <tbody>
-    <tr id=placeholder-report>
-      <th><a href=scheme://host:port/path/to/report>Thoughts on placeholders</a>
-      <td>Place Holder, Place Holder, and Place Holder
-    </tr>
-  </tbody>
-</table>
--->
+<p>This list will be kept <a href=#amendments>updated</a> by the Chairs
+to reflect the current set of Work Items of the Community Group.
+
+<p>The Chairs may add Work Items, but must not add Work Items which lack
+the support of at least two <a href=#implementer>implementers</a>.
+
+<p class=note>Typically, Work Items begin life as a Proposal before
+being adopted.
+
+<p>When a Work Item's Editors deem the Work Item ready for migration to
+a standards body, they will notify the Chairs. The Editors and Chairs
+will then work together to identify the best destination, and will work
+with the destination standards body to successfully migrate the work.
+
+<p>The Chairs may remove Work Items. The Chairs must announce the
```

Comment:
What forum should folks watch to get these announcements?

Reply:
Yeah, we should probably spell that out. Good catch. I've added text for this in 7e59061.